cmake/std/atdm/utils/get_known_system_info.sh

```diff
@@ -112,9 +112,9 @@ elif [[ $realHostname == "white"* ]] ; then
 elif [[ $realHostname == "ride"* ]] ; then
   hostnameMatch=ride
   hostnameMatchSystemName=ride
-elif [[ $realHostname == "waterman"* ]] ; then
-  hostnameMatch=waterman
-  hostnameMatchSystemName=waterman
+elif [[ $realHostname == "weaver"* ]] ; then
+  hostnameMatch=weaver
+  hostnameMatchSystemName=weaver
```

Comment:
This also needs to be deleted.  Custom configurations don't get listed is this file.  We really just want to delete the old 'waterman' configuration in this file and other files.

Reply:
@bartlettroscoe I tried deleting the option but ran into this configuration error:

"Error, could not determine a system configuration for hostname='weaver11', aborting env loading script!"

I reintroduced these lines to the PR, but can modify in another way that you think appropriate